src/dmd/id.d

```diff
@@ -314,6 +314,7 @@ immutable Msgtable[] msgtable =
     { "dup" },
     { "_aaApply" },
     { "_aaApply2" },
+    { "__tmpFunc"},
```

Comment:
Would be nice to give a bit of a description here.

Reply:
temporary function used as mock-up scope to evaluate function parameters ;)

Reply:
I know that _now_, I might not in 6 months time.